lib/src/components/NxToast/NxToast.scss

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+.nx-toast__wrapper {
+  ///nx-toast__wrapper has position absolute to ensure toasts stay in the top right corner of the page.
+  //Using display:flex and flex-end ensures the right side of the toast is aligned with the right side of 
+  //nx-page-main. The nx-toast_container has position fixed applied so it remains fixed while scrolling.
+  position: absolute;
+  top: 0;
+  right: 0;
+  display: flex;
+  justify-content: flex-end;
+  margin-right: var(--nx-spacing-6x);
+}
+
+.nx-toast__container {
+  position: fixed;
+  width: 500px;
+}
+
+.nx-toast {
+  position: relative;
+  animation: animationIn 0.4s linear;
+
+  &.nx-toast--closing {
+    animation: animationOut 0.4s linear;
+  }
+  
+  .nx-alert {
+    margin: var(--nx-spacing-2x) 0 0 0;
+  }
+}
+
+@keyframes animationIn {
+  0% {
+    margin-top: 0;
+    max-height: 0px;
+    right: -500px;
+    opacity: 0;
+  }
+  40% {
+    margin-top: var(--nx-spacing-2x);
+    right: -500px;
+    opacity: 0;
+    max-height:74px;
+  }
+  100% {
+    right:0;
+    opacity: 1;
+  }
+}
+
+@keyframes animationOut {
+  0% {
+    max-height: 75px;
+    right: 0;
+    opacity: 1;
+  }
+  75% {
+    max-height:74px;
+    right: -500px;
+    opacity: 0;
+  }
+  100% {
+    max-height: 0px;
+    right:-500px;
+    opacity: 0;
+  }
+}
+
+/*
+  If the user has expressed their preference for reduced motion,
+  then only animate the fade effect.
+*/
+@media (prefers-reduced-motion: reduce) {
+  .nx-toast {
+    animation: reducedAnimationIn 0.4s linear;
+
+    &.nx-toast--closing {
+      animation: reducedAnimationOut 0.4s linear;
+    }
+  }
+}
+
+@keyframes reducedAnimationIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
+@keyframes reducedAnimationOut {
+  0% {
+    opacity: 1;
+  }
```

Comment:
Like with the other ones, we don't need the in animation's end point or the out animation's start point.

Reply:
> Like with the other ones, we don't need the in-animation's end point or the out-animation's start point.

Removed here initially: [ca49163](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/ca49163b4dce7eee38131b2d3ca3b6652eff2c44), and then animation fixed here: [df8c159](https://github.com/sonatype/sonatype-react-shared-components/pull/688/commits/df8c159d9b7fd58ffcf0433f639aee0221677fd5)